### DISCUSSION TITLE
Question marks in markdown with Markdown lang extra

### DISCUSSION THREAD

**Discussion by Demianeen** [LazyVim/LazyVim#2255] [Q&A] [answered]
After enabling Markdown extra lang I see question marks in Markdown:
![CleanShot 2023-12-26 at 09 53 40@2x](https://github.com/LazyVim/LazyVim/assets/51330172/e90907b4-c606-478d-9a9d-886d3d149e4a)

I use Hack Nerd Font so it should supply all the icons. Do I need to install some special font for the headlines.nvim not to show question marks?

**Comment by dpetka2001** [comment, marked as answer]
Unfortunately, it's a problem with your font. See this [comment](https://github.com/lukas-reineke/headlines.nvim/issues/41#issuecomment-1556334775).

**Reply by Demianeen** [reply]
Wow! Thanks! Never thought there was a difference between different Nerd fonts
